docs/LOGGING.md

```diff
@@ -30,28 +30,49 @@ Logs coming from xen-tools container are of three kinds:
   * ```qemu-dm-[VM_NAME]``` logs the qemu device model output
   * ```qdisk-[VM ID]``` logs the qdisk output
 
-If there are too many logs coming from various containers and the log reader is not able to keep up, oldest logs from the current circular buffer will be overwritten. All logs read by memlgod by rsyslogd will be written to disk queues. When there is heavy disk usage and rsyslogd does not get sufficient time to write incoming logs to disk, it can result in log loss from memlogd.
+All logs from memlgod and from /dev/kmsg read by newlogd will be written to disk log file and then to be compressed into gzip log files. When there is heavy disk usage or CPU load and newlogd does not get sufficient time to write incoming logs to disk, it can result in log loss from memlogd.
```

Comment:
typo "memlgod"

Reply:
Can we also have log loss if there is significant load of logs for a sustained amount of time and the network is slow or down? We'll hit some disk space limit at some point in time.

Reply:
When we have those losses can we detect that there was a loss event? Know how many log messages were lost (1 vs. 1000)?

Reply:
ok. yes, as mentioned above, if log is overwhelming, the device CPU is too high to process, I can image the memlogd queues can be overwritten. I actually tried to see if we can detect this, but I can not find a way. But with my test turned on full debug mode, and i don't see this issue on my device. the memlogd has 5000 lines, so there is a chance, but the chance is small.